### ISSUE TITLE
Before I try and help you.

### ISSUE THREAD

**Issue by DukeVenator** [A3Antistasi/A3-Antistasi#163]
Well for one having all your variables in spanish makes working with your code a little hard.
The 2nd is the lack of credit being included 
A good example is the jamming script which can be found here 
https://forums.bohemia.net/forums/topic/203810-release-radio-jamming-script-for-task-force-radio/?page=3

I checked the script and found no credit attach even though it was included by default with the original script.  

That right there is a big concern! So if I help you with new features and then over time you wipe my name from the code would that count as intellect theft?

Another annoyance is the lack of comments that would help people in the future understand what this section of code does! 

Also the way the jamming script was modified could be improved

**Comment by DukeVenator** [comment]
[_entity]execVM "RadioJammer\jammer_Check.sqf";  

Which then leads to a script which does jamming check 

which allows me to do all sorts of fun stuff with out editing the original jamming code while allowing a infinite number of generated jammers that can be disabled on the fly!

**Comment by Barbolani77** [comment]
Hi!

About the spanish variable names, well, not a big help definetly, but in future I will try to remember to use english.

About the creds of the Jam script: have you checked the credit section in briefing? It is not only mentioned, but also if you check in the BIS forum thread I ask permission to the author to include it.

No third party script in Antistasi should be without creds, if you spot something, let me know.

About the script itself: have you seen the script in Antistasi? It's an adaptation of that which will allow use RT as jammers for other factions which not own the RT.

Totally untested and with zero feedback ATM. When I released this, my PC crashed and I managed to repair it yesterday.

The lack of comments, yes, I know, I am not a pro coder and all this has been done by myself, but again, I will implement comments in future.

Coming back to the jamming script, first of all some testing is needed, and when it works we may think about other ways to improve.
